Mathlib/Data/ENNReal/Operations.lean

```diff
@@ -308,6 +314,7 @@ theorem sub_top : a - ∞ = 0 := WithTop.sub_top
 -- Porting note: added `@[simp]`
 @[simp] theorem sub_eq_top_iff : a - b = ∞ ↔ a = ∞ ∧ b ≠ ∞ := WithTop.sub_eq_top_iff
 
+@[aesop (rule_sets := [finiteness]) safe apply]
```

Comment:
This also isn't `safe`, as it's possible that `a = b =  ∞`. I think think can also be `unsafe 75%`.